data/installed-tests/fwupdmgr-p2p.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# ---
+echo "Starting P2P daemon..."
+FWUPD_DBUS_SOCKET=/var/run/fwupd.sock /usr/libexec/fwupd/fwupd --verbose --timed-exit --no-timestamp &
+sleep 5
+
+# ---
+echo "Starting P2P client..."
+FWUPD_DBUS_SOCKET=/var/run/fwupd.sock /usr/bin/fwupdmgr get-devices --json
```

Comment:
so I was kinda just thinking that you could do something like:
FWUPD_DBUG_SOCKET=foo `/usr/share/installed-tests/fwupdmgr`

Then it proves all the existing installed tests work in socket mode or in real dbus mode.

Reply:
I don't think we need to test everything; if the p2p get-devices works then that's already ~6 DBus messages and signals. I think having two tests (one of which only runs as root) means it's a lot easier to debug failures.